cfgov/cfgov/urls.py

```diff
@@ -213,6 +214,12 @@
     ],
         namespace='transcripts')),
 
+    # TODO: Remove prototype landing page routes when all organisms and molecules have been implemented elsewhere.
```

Comment:
cc @kurtw @kave Would `if settings.DEBUG` work in `cfgov/cfgov/urls.py` to limit those routes to the development environment?